### PR TITLE
Created an API that shows available tokens for translation

### PR DESCRIPTION
**Pull request by Joel-C-Johnson**
### Added Two New API
    1) it shows the available tokens for translation.
            Output : {"GEN" : [tokens], "EXO" : [tokens], "LEV" : [tokens]}
    2) Its shows the number of tokens (counts) available for transaltion
           Output : {"GEN" : 5, "EXO" : 21, "LEV" :0 }

### CODE REVIEW

File: mt2414/main.py [hunk 1]
```diff
@@ -369,11 +373,14 @@ def availablesbooks():
     cursor = connection.cursor()
     cursor.execute("SELECT st.book_name, st.revision_num FROM sources s LEFT JOIN sourcetexts st ON st.source_id = s.id WHERE s.language = %s AND s.version = %s",(language, version))
     books=[]
-    al = cursor.fetchall()
-    for rst in range(0, len(al)):
-        books.append(al[rst])
-    return json.dumps(books)
-    cursor.close()
+    if al:
+        al = cursor.fetchall()
```
Comment (faithfulmaster) [inline mt2414/main.py]: This line should come before the if statement. Right?

Comment (Joel-C-Johnson) [inline mt2414/main.py]: yes

File: mt2414/main.py [hunk 2]
```diff
@@ -148,13 +148,18 @@ def reset_password2(code):
...
+    if len(rst) > 0:
+        email = rst[0]
+        password_salt = str(uuid.uuid4()).replace("-","")
+        password_hash = scrypt.hash(password, password_salt)
+        cursor.execute("UPDATE users SET verification_code = %s, password_hash = %s, password_salt = %s, updated_at = current_timestamp WHERE email = %s", (None, password_hash, password_salt, email))
+        cursor.close()
+        connection.commit()
+        return '{"success":true, "message":"Password has been reset"}\n'
+    else:
+        return '{"success":false, "message":"Failed to reset password"}\n'
```
Comment (faithfulmaster) [inline mt2414/main.py]: Why is there a \n at the end of the return statement?

File: mt2414/main.py [hunk 3]
```diff
@@ -148,13 +148,18 @@ def reset_password2(code):
...
-    connection.commit()
-    return '{"success":true, "message":"Password has been reset"}\n'
+    if len(rst) > 0:
+        email = rst[0]
+        password_salt = str(uuid.uuid4()).replace("-","")
+        password_hash = scrypt.hash(password, password_salt)
+        cursor.execute("UPDATE users SET verification_code = %s, password_hash = %s, password_salt = %s, updated_at = current_timestamp WHERE email = %s", (None, password_hash, password_salt, email))
+        cursor.close()
+        connection.commit()
+        return '{"success":true, "message":"Password has been reset"}\n'
```
Comment (faithfulmaster) [inline mt2414/main.py]: Why is there a \n at the end of the return statement?

**Pull request merged**